sfs/array.py

```diff
@@ -84,7 +84,7 @@ def as_secondary_source_distribution(arg, **kwargs):
     return SecondarySourceDistribution(x, n, a)
 
 
-def linear(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
+def linear(N, spacing, *, center=[0, 0, 0], orientation=[1, 0, 0]):
```

Comment:
What does the `*` do here?

Reply:
That's the notation for keyword-only arguments, see https://www.python.org/dev/peps/pep-3102/.

Reply:
OK, I see, but why do we need this already, even `linear` has no variable arguments?
If we introduce them later, we could still add `*`, couldn't we?

Reply:
Do I get it right and `*` forces the keyword arguments to be used only as keywords?
So you have to write `def linear(N, spacing, center=[0, 1, 0])` and you are not allowed to write `def linear(N, spacing, [0, 1, 0])`.
Then I get the point.

Reply:
> forces the keyword arguments to be used only as keywords

Exactly. That's what "keyword-only" means.

> you are not allowed to write `linear(N, spacing, [0, 1, 0])`

Exactly.

And if we introduce them later, it will be a breaking change.
If we decide to remove them later, it's no problem, because removing them is backwards-compatible.

Reply:
:+1: